### PR TITLE
Added mapi package from CRAN

### PR DESCRIPTION
**Pull request by spiry34**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by spiry34** [comment]
Wow, I'm feeling lucky :-)

File: recipes/r-mapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  summary: Mapping Averaged Pairwise Information (MAPI) is an exploratory method providing graphical
+    representations summarizing the spatial variation of pairwise metrics (eg. distance,
+    similarity coefficient, ...) computed between georeferenced samples.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - spiry34
```
Comment (dbast) [inline recipes/r-mapi/meta.yaml:65]: Please also add the conda-forge/r team, that helps to maintain and update all the r recipes.
```suggestion
    - conda-forge/r
    - spiry34
```

Comment (spiry34) [reply recipes/r-mapi/meta.yaml:66]: Thanks @dbast , this is done!

File: recipes/r-mapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - r-sf >=0.5
+
+test:
+  commands:
+    - $R -e "library('mapi')"           # [not win]
+    - "\"%R%\" -e \"library('mapi')\""  # [win]
+
+about:
+  home: https://www1.montpellier.inra.fr/CBGP/software/MAPI/
+  license: GPL-2.0-or-later
```
Comment (dbast) [inline recipes/r-mapi/meta.yaml:55]: The cran meta data says `License: GPL (>= 3)`, so:
```suggestion
  license: GPL-3.0-or-later
```

**Comment by dbast** [comment]
@spiry34 Thanks and welcome to conda-forge!

**Pull request merged**